src/handlers/testing/align/components/GradeTestCaseResultForm.tsx

```diff
@@ -0,0 +1,113 @@
+import { Box, Text } from 'ink';
+import SelectInput from 'ink-select-input';
+import TextInput from 'ink-text-input';
+import { useState } from 'react';
+import type { AsyncHandlerOf } from '../../../../util/types';
+import { AfterGradeAction } from '../util/models';
+import { AfterGradeSelectAction } from './AfterGradeSelectAction';
+import { Question } from './Question';
+
+enum BinaryGrade {
+  ACCEPTED = 'accepted',
+  REJECTED = 'rejected',
+}
+
+export function GradeTestCaseResultForm(props: {
+  body: unknown;
+  output: unknown;
```

Comment:
Is there any more specific type that we can use for these? I see we are stingifying body below, could this be Record<string, unknown> ?